rest/routes.go

```diff
@@ -568,8 +568,11 @@ func (s *Service) fetchUserCertKey(rw http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	if s.Depot == nil {
```

Comment:
Is this supposed to do something or is it handled by the depot-checking middleware?

Reply:
I forgot to remove it, done.